config/environments/production.rb

```diff
@@ -23,9 +23,9 @@
   config.serve_static_assets = false
 
   # Compress JavaScripts and CSS.
-  #config.assets.js_compressor = :uglifier
-  config.assets.js_compressor = Uglifier.new(:mangle => false)
-  # config.assets.css_compressor = :sass
+  config.assets.js_compressor = :uglifier
```

Comment:
Hmm... sorry. I don't think I was done with that issue yet I suppose. It was suppose to enable compression by uncomment .css_compressor but I think there was an that happened in doing so.